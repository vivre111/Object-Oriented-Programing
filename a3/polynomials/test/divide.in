create a -9/2 8 4/3 4 2/1 2 0/5 1
create b 3/5 0
create c 0/2 12
create d 9/-2 12
divide c d
print c
print d
divide a b
divide a d 
divide a d 
divide c b
divide a a
divide b a

