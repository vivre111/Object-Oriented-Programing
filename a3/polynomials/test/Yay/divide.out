Quotient: 0
Remainder: 0
0
(-9/2)x^12
Quotient: (-15/2)x^8 + (20/9)x^4 + (10/3)x^2
Remainder: 0
Quotient: 0
Remainder: (-9/2)x^8 + (4/3)x^4 + (2)x^2
Quotient: 0
Remainder: (-9/2)x^8 + (4/3)x^4 + (2)x^2
Quotient: 0
Remainder: 0
Quotient: (1)
Remainder: 0
Quotient: 0
Remainder: (3/5)
